src/util/file_reader.ts

```diff
@@ -40,8 +40,9 @@ async function writeUrlContentsToFile(
   url: string,
   pathPrefix: string,
   pathDefaultExt: string,
+  useProxy: boolean = true,
```

Comment:
```suggestion
  useProxy: boolean = false,
```

I think we should actually just default this to false - this method is used to fetch data before the crawl starts, so should really not need to use the proxy here, but maybe good to leave as option for later.